convert2rhel/checks.py

```diff
@@ -422,10 +422,21 @@ def _bad_kernel_version(kernel_release):
 
 def _bad_kernel_package_signature(kernel_release):
     """Return True if the booted kernel is not signed by the original OS vendor, i.e. it's a custom kernel."""
-    kernel_pkg = run_subprocess(
+    kernel_pkg, return_code = run_subprocess(
         ["rpm", "-qf", "--qf", "%{NAME}", "/boot/vmlinuz-%s" % kernel_release], print_output=False
-    )[0]
+    )
     logger.debug("Booted kernel package name: {0}".format(kernel_pkg))
+
+    vmlinuz_path = "/bootvmlinuz-%s" % kernel_release
```

Comment:
```suggestion
    vmlinuz_path = "/boot/vmlinuz-%s" % kernel_release
```

Reply:
Thank you for pointing out this typo, I'll go ahead and get this fixed!